kratos/processes/apply_ray_casting_process.cpp

```diff
@@ -65,7 +65,7 @@ namespace Kratos
 	void ApplyRayCastingProcess<TDim>::Execute()
 	{
         if(mIsSearchStructureAllocated) // we have not initialized it yet
-            mpFindIntersectedObjectsProcess->Initialize();
+            mpFindIntersectedObjectsProcess->ExecuteInitialize();
```

Comment:
`Initialize` and `ExecuteInitialize` have exactly the same behavior right?

Reply:
Yes